nameko_stripe/handlers/event.py

```diff
@@ -0,0 +1,131 @@
+import json
+import logging
+from collections import defaultdict
+from fnmatch import fnmatch
+from functools import partial
+
+from eventlet.event import Event
+from nameko.exceptions import serialize
+from nameko.web.server import WebServer
+from werkzeug.exceptions import BadRequest, HTTPException
+from werkzeug.routing import Map, Rule
+from werkzeug.wrappers import Request, Response
+
+
+from nameko_stripe.handlers.webhook import WebhookHandler
+
+
+logger = logging.getLogger(__name__)
+
+
+class Router:
+    def __init__(self, container, handlers=None):
+        self.container = container
+        self.handlers = handlers or set()
+
+    def __call__(self, request):
+        return self.handle(request)
+
+    def handle(self, request):
+        events = set()
+        event_name = self.parse_event_name(request)
+        for handler in self.handlers:
+            if fnmatch(event_name, handler.event_name):
+                event = Event()
+                events.add(event)
+                handler.handle_request(request, event)
+        [event.wait() for event in events]
+
+    def parse_event_name(self, request):
+        data = request.data.decode("utf8")
```

Comment:
You're using `"utf8"` both here and in `webhook.py`. What about moving it to `constants.py`?

Reply:
I don't see how that would help .) "utf8" is a pretty standard value, how would `UTF8 = "utf8"` or `ENCODING = "utf8"` in constants help? It seems it would only making things more complicated ;)

Reply:
My suggestion here was to have a `UTF8 = "utf8"` `constants.py` that is used in all the places where we need to use the `"utf8"` string.

It's not to improve readability in any way and this it's just 4 characters, which makes it easy to read, but, as per other string constants, we would have a central place to define this value and would enforce a consistent way of doing it (even preventing defining it differently in other places in the future: `'UTF8'`, `'UTF-8'`, `'utf-8'`...)

Also, even if we have tests for our code, it would prevent potential mistakes when writing the string: if you succeed importing the `UTF8` constant then you know the string value is going to be correct.

I'm applying the same reasoning here as per other constants that we may have.